SU2_CFD/include/numerics/CNumerics.hpp

```diff
@@ -136,6 +136,14 @@ class CNumerics {
   const su2double
   *TurbPsi_i,  /*!< \brief Vector of adjoint turbulent variables at point i. */
   *TurbPsi_j;  /*!< \brief Vector of adjoint turbulent variables at point j. */
+  su2double lengthScale_i, lengthScale_j; /*!< \brief length scale for SST */
+  su2double FTrans;   /*!< \brief SAS function */
+  su2double Q_SAS_1;   /*!< \brief SAS function */
+  su2double Q_SAS_2;   /*!< \brief SAS function */
+  su2double L;   /*!< \brief SAS function */
+  su2double L_vK_1;   /*!< \brief SAS function */
+  su2double L_vK_2;   /*!< \brief SAS function */
```

Comment:
If these were here for debugging, please use local variables in the function where these quantities are used.